package.json

```diff
@@ -58,27 +58,26 @@
     }
   },
   "devDependencies": {
-    "benchmark": "1.0.0",
+    "benchmark": "^2.1.4",
     "browserify": "^16.2.3",
     "chai": "^4.2.0",
     "eslint": "^6.1.0",
     "eslint-find-rules": "^3.3.1",
     "eslint-plugin-html": "^6.0.0",
     "eslint-plugin-jsdom-internal": "link:./scripts/eslint-plugin",
     "js-yaml": "^3.13.1",
-    "karma": "^1.7.1",
-    "karma-browserify": "^5.3.0",
-    "karma-chrome-launcher": "^2.2.0",
+    "karma": "^4.3.0",
+    "karma-browserify": "^6.1.0",
+    "karma-chrome-launcher": "^3.1.0",
     "karma-mocha": "^1.3.0",
-    "karma-mocha-webworker": "^1.3.0",
```

Comment:
Do we still run the tests in a web worker?

Reply:
On master, yes. But here I removed it.